xds/src/main/java/io/grpc/xds/AbstractXdsClient.java

```diff
@@ -470,14 +470,9 @@ private void handleRpcStreamClosed(Status error) {
         // has never been initialized.
         retryBackoffPolicy = backoffPolicyProvider.get();
       }
-      long delayNanos = 0;
-      if (!responseReceived) {
-        delayNanos =
-            Math.max(
-                0,
-                retryBackoffPolicy.nextBackoffNanos()
-                    - stopwatch.elapsed(TimeUnit.NANOSECONDS));
-      }
+      long delayNanos = Math.max(
+          0,
+          retryBackoffPolicy.nextBackoffNanos() - stopwatch.elapsed(TimeUnit.NANOSECONDS));
```

Comment:
I assume the default backoff policy is what caps it to 1 second, right?

Reply:
Yeah. https://github.com/grpc/grpc-java/blob/72ae95792c79ffe2641e4e697d4802280edde39a/core/src/main/java/io/grpc/internal/ExponentialBackoffPolicy.java#L40

Reply:
(the description says it's capped at 1 QPS)